src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, isMarried, partnerName, sex } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age && <p className="Person__age">I am {age}</p>}
+      {isMarried === true ? (
+        <p className="Person__partner">
+          {partnerName} is my {sex === 'm' ? 'wife' : 'husband'}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
+    </section>
```

Comment:
1) u don't need to write `isMarried === true`.It understands itself that this is a boolean variable.
2) refactore ur code for better readability
```suggestion
    <section className="Person">
      <h2 className="Person__name">
        My name is {name}
      </h2>
      
      {age && (
        <p className="Person__age">I am {age}</p>
      )}
      
      {isMarried ? (
        <p className="Person__partner">
          {partnerName} is my {sex === 'm' ? 'wife' : 'husband'}
        </p>
      ) : (
        <p className="Person__partner">I am not married</p>
      )}
    </section>
```